### ISSUE TITLE
Select Issues/FR for QHack2023

### ISSUE THREAD

**Issue by DhruvaSambrani** [QuantumBFS/Yao.jl#453]
[QHack](https://github.com/XanaduAI/QHack2023) allows participants to submit a PR for a new feature to any open source project. As the defacto Quantum Computing package for julia, there may be interest in contributing to the Yao ecosystem. Thus, it might be prudent to direct potential participants to issues or FRs which need to be implemented. Preferably, we could set up a website with a small description of what and how we want features to be implemented, so we can benefit more from contributors.

**Comment by Roger-luo** [comment]
Hi thanks for your interest, we are currently putting this version under maintenance mode and hopefully deliver a refactored version next year. So it's always welcome to submit bug fixes or small feature implementations (as listed in the issue), you can pick any of them or anything you think is cool and propose that.

**Comment by DhruvaSambrani** [comment]
`EasyBuild` might be a good place for people to contribute? Even if there is internal refactors, APIs shouldn't change so drastically that such high level things require changes?

**Comment by Roger-luo** [comment]
yeah, `EasyBuild` is a good place to contribute, the surface API won't change much, and at least easy to transfer from.

**Comment by GiggleLiu** [comment]
The QHack 2023 coordination question was answered in the discussion: small fixes and features were welcome, and `EasyBuild` was identified as the stable contribution area while the larger refactor proceeded. The event-specific request has concluded, so closing this completed planning thread.
